modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PostalAddressResourceTest.java

```diff
@@ -96,16 +124,66 @@ public void testDeletePostalAddress() throws Exception {
 	@Override
 	@Test
 	public void testPatchPostalAddress() throws Exception {
-		super.testPatchPostalAddress();
+		PostalAddress postPostalAddress =
```

Comment:
leave the super.testPatchPostalAddress() and move this logic in a _testPatchPostalAddressWithSubtype()